vtex-apps/order-quote
# [ORDERQUOTE-7] Display totalizers horizontally when printing

#### What does this PR do? \*
Formats totalizers horizontally when printing. Update dependency `vtex.css-handles` to major 1.x.

#### How to test it? \*
Test in workspace [annaorderquote](https://annaorderquote--sandboxusdev.myvtex.com/). Create order quote and click Print to see file.

_Before_
<img width="770" alt="Screen Shot 2022-06-07 at 4 11 32 PM" src="https://user-images.githubusercontent.com/53097865/172474078-364be42e-5770-47d3-a64f-2c0f96974340.png">


_After_
<img width="756" alt="Screen Shot 2022-06-07 at 4 12 39 PM" src="https://user-images.githubusercontent.com/53097865/172474107-ae62a3f6-97c4-424d-8c70-749d235d8e96.png">
